Class-Code/Elementary-CA/ghouston/sketch.js

```diff
@@ -1,29 +1,53 @@
-// Gordon Houston
-// Elementary CA Project
-// 2/28/19
-
-// Class Project
-
-let gen1 = Array(57).fill(0);
+const ruleset = [0,1,1,1,1,1,1,0]
+let ca = [];
+let gensize = 101;
 let gridsize;
+
 function setup() {
   createCanvas(400, 400);
-  gen1[round(gen1.length / 2)] = 1;
-  gridsize = width / gen1.length;
-  console.log(gridsize);
+  let gen0 = Array(gensize).fill(0);
```

Comment:
a well-placed comment or two could go a long way with making this section a little clearer.